client-src/elements/chromedash-textarea_test.ts

```diff
@@ -9,7 +9,7 @@ describe('chromedash-textarea', () => {
     );
     assert.exists(component);
     assert.instanceOf(component, ChromedashTextarea);
-    const textareaElement = component.shadowRoot.querySelector('textarea');
+    const textareaElement = component.shadowRoot!.querySelector('textarea');
```

Comment:
See if using `renderRoot` avoids the need for `!`.
```suggestion
    const textareaElement = component.renderRoot.querySelector('textarea');
```